src/server/cli/test-runner.js

```diff
@@ -196,7 +197,12 @@ async function getTestRunnerOptions(argv = []) {
 	};
 }
 
+function installDeps() {
+	execSync('npx playwright install-deps', { stdio: 'pipe' });
```

Comment:
I played around with the async version of this and wrapped it in a Promise, but marshalling the stdout/err around seemed way more complicated than it was worth. Open to other suggestions though.